src/content/cloud/external-resources.mdx

```diff
@@ -0,0 +1,224 @@
+
+All of this information can be found and in a more concise format in our [References section](/reference/manifest/#external-object-optional).
+
+### Using dynamic endpoints with external resources
+
+[Dynamic Endpoints](/reference/manifest/#use-dynamic-endpoints-for-an-external-url) is an Okteto feature that lets you define non-static URLs in your Okteto manifest. There are generally three endpoint scenarios:
+
+1. A static, unchanging URL that you know and that will remain the same across all users and development environments
+1. A hybrid URL that may have a static component but incorporate a dynamic one such as a username (e.g. user1.example.com/endpoint)
+1. A fully dynamic URL that my be completely unique per user
+
+Using the Dynamic Endpoints feature Okteto provides means you can omit the `url:` key from a specific external resource in your Okteto manifest and instead compile that URL within the `deploy` section (either directly or within an included script). 
+
+For example, if have the following `deploy` section defined:
+
+```yaml
+deploy:
+  image: okteto/pipeline-runner:1.0.0-sam
+  commands:  
+  - name: Create an AWS SQS queue
+    command: |
+      sh sqs/deploy.sh
+```
+
+We can define the endpoint dynamically within the `deploy.sh` script ([example here](https://github.com/okteto/external-resources-aws/blob/main/sqs/deploy.sh)). In simpler terms, the URL is dynamically compiled on this line:
+
+```yaml
+dashboard="https://${AWS_REGION}.console.aws.amazon.com/sqs/v2/home?region=${AWS_REGION}#/queues/${encoded}"
+```
+
+That will compile the appropriate URL, using Okteto environment variables, that specify where a developer in the same namespace will go when they click on the AWS SQS external resource component in the Okteto UI (see below).
+
+<p align="center"><Image src="/docs/cloud/images/external_resources_sqs_demo.png" alt="AWS SQS external resource demo UI component" width="450" /></p>
+
+## Example: Deploy a Development Environment with External Resources
+
+Now that we've walked through a sample of an `external` configuration, let's look at using this within the broader context of an application we're working on. You can navigate to our [demo repo](https://github.com/okteto/external-resources-aws) to follow along if you'd like.
+
+In this example we'll be configuring and deploying a development environment that includes polyglot microservices, an AWS SQS queue, and an S3 bucket.
+
+<p align="center"><Image src="/docs/cloud/images/external_resources_architecture.png" alt="Application architecture diagram" width="450" /></p>
+
+### Prerequisites
+
+1. Okteto CLI 2.14 or newer
+1. An AWS account
+1. An Okteto account
+1. Create a set of IAM keys for your AWS account (If you are using Okteto Self-Hosted, you can directly assign an AWS Role)
+1. Create the following Okteto secrets:
```

Comment:
Link to secrets?

Reply:
Added in latest commit: 58b9d8f90c5afaaaf3ce9eadc19edf63560a3a96